### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by JasonRShaver**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,50 +1,114 @@
...
 Installation
-============
+===============
+
+For OSX and Linux, you can install with the following curl script.
+
+.. code-block:: console
 
-For installation steps for common platforms, please take a look at our `preview installation guide <http://github.com/Azure/azure-cli/blob/master/doc/preview_install_guide.md>`__.
+   $ curl -L https://aka.ms/ProjectAzInstall | sudo bash
```
Comment (derekbekoe) [inline README.rst]: Will we still have the same install URL?

File: README.rst [hunk 2]
```diff
@@ -1,50 +1,114 @@
...
+   $ az vm show -g WebPropertiesRG -n Bizlogic
 
+Querying
+^^^^^^^^
+
+You can use the ``--query`` parameter and the JMESPath query syntax to customize your output.
+
+.. code-block:: console
+
+   $ az vm list --query '[].{name:name,os:storageProfile.osDisk.osType}'
```
Comment (derekbekoe) [inline README.rst]: All the examples use table format but table format isn't mentioned.

Consider adding `-otable` to the commands so it's obvious that you're using table format.
Or, have a section describing the table formats and say 'the example below use table format'.

Thoughts?

File: README.rst (hunk 1 above)
Comment (JasonRShaver) [inline README.rst]: Thanks, fixed

File: README.rst (hunk 2 above)
Comment (JasonRShaver) [inline README.rst]: Good call, fixed

**Pull request merged**